collections/_priorities/physics-01-adapt-the-way-godot-exposes-physics-to-mirror-jolt.md

```diff
@@ -0,0 +1,16 @@
+---
+category: physics
+rank: 1
+anchor: "adapt-the-way-godot-exposes-physics-to-mirror-jolt"
+title: "Adapt the way Godot exposes physics to mirror Jolt"
+description: |
+  As much as our nodes are made to be compatible with multiple physics engines, the existing integration of Jolt (via the [godot-jolt](https://github.com/godot-jolt/godot-jolt) add-on) is not optimal, as there are numerous features that can’t be implemented in Godot due to the current way the system works. In addition to integrating Jolt as the default 3D physics engine, we want to modernize our node bindings in order to fully exploit the new library.
```

Comment:
See https://github.com/godotengine/godot-proposals/issues/7400 and https://github.com/godotengine/godot-proposals/issues/7401 as two actionable items for this. Implementing these proposals will improve Godot's compatibility with both Jolt physics and glTF physics, good for the asset pipeline.